src/layout/components/Sidebar/SidebarLogo.vue

```diff
@@ -69,6 +69,29 @@ export default class extends Vue {
   @Prop({ required: true }) private collapse!: boolean
 
   private title = 'APISIX'
+
+  get getIco() {
```

Comment:
We could use an elegant way to fix this issue... Let me check.

Reply:
sure thisPR is too complicated, but I think use the same link from link.rel will keep the Icon link consistent

Reply:
I'd love to know if the latest version of dashboard is working on your side?